src/Psy/Input/FilterOptions.php

```diff
@@ -135,7 +135,7 @@ private function validateRegex($pattern)
         set_error_handler(['Psy\Exception\ErrorException', 'throwException']);
         try {
             preg_match($pattern, '');
-        } catch (ErrorException $e) {
+        } catch (\ErrorException $e) {
```

Comment:
i think this one is supposed to be `Psy\Exception\ErrorException`, so we're missing a `use` statement at the top, not a slash here.